.php-cs-fixer.php

```diff
@@ -1,61 +1,44 @@
 <?php
 
-// $header = <<<'EOF'
-// @link https://github.com/organisation
-// @copyright Organisation
-// @license MIT License
-// EOF;
-
 use PhpCsFixer\Config;
 use PhpCsFixer\Finder;
 
 $rules = [
     /*
-     * A variant of Symfony RuleSet that isn't too far from Laravel's StyleCI config
-     * Use PHPStorm's Default and adapt instead of using "Laravel" / "PSR12" / "Symfony 2", those are far too greedy.
-     *
      * RuleSet cascades/overrides:
      * @PhpCsFixer (risky)  https://raw.githubusercontent.com/FriendsOfPHP/PHP-CS-Fixer/2.18/src/RuleSet/Sets/PhpCsFixerSet.php
      *      @Symfony        https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/RuleSet/Sets/SymfonySet.php
      *          @PSR12      https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/RuleSet/Sets/PSR12Set.php
      *              @PSR2   https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/RuleSet/Sets/PSR2Set.php
-     *
-     * Laravel              https://gist.github.com/laravel-shift/cab527923ed2a109dda047b97d53c200
      */
     '@Symfony' => true,
 
     // @Symfony overrides
+    'blank_line_before_statement' => false,
     'concat_space' => ['spacing' => 'one'],
-    'phpdoc_summary' => false,
-    'phpdoc_align' => false,
-    'single_line_throw' => false,
-    'yoda_style' => false,
-
-    // @Symfony overrides: Laravel
+    'echo_tag_syntax' => false,
     'increment_style' => ['style' => 'post'],
+    'no_alternative_syntax' => false,
     'no_extra_blank_lines' => [
         'tokens' => [
             'extra',
             'throw',
             'use',
         ],
     ],
+    'not_operator_with_space' => false,
+    'phpdoc_align' => false,
+    'phpdoc_summary' => false,
+    'single_line_throw' => false,
+    'single_line_comment_spacing' => false,
```

Comment:
Disabled that new rule as it would unnecessarily touch 90 files:
`//commet` -> `// comment`.
We should consider removing all those debug comments instead.